orchestrator/test_runner/src/main.rs

```diff
@@ -75,9 +76,9 @@ lazy_static! {
 /// this value reflects the contents of /tests/container-scripts/setup-validator.sh
 /// and is used to compute if a stake change is big enough to trigger a validator set
 /// update since we want to make several such changes intentionally
-pub const STAKE_SUPPLY_PER_VALIDATOR: u128 = 1000000000000000000000;
+pub const STAKE_SUPPLY_PER_VALIDATOR: Uint256 = u256!(1000000000000000000000);
 /// this is the amount each validator bonds at startup
-pub const STARTING_STAKE_PER_VALIDATOR: u128 = STAKE_SUPPLY_PER_VALIDATOR / 2;
+pub const STARTING_STAKE_PER_VALIDATOR: Uint256 = u256!(500000000000000000000);
```

Comment:
Why did you change to the fixed constant value instead of half of STAKE_SUPPLY_PER_VALIDATOR?